.github/workflows/test_production_build.yml

```diff
@@ -0,0 +1,78 @@
+name: Test production build
+
+on: push
+
+env:
+  APP_NAME: rails_templates
+  DOCKER_IMAGE: ${{ github.repository }}
+  DOCKER_REGISTRY_HOST: ${{ secrets.DOCKER_REGISTRY_HOST }}
+  RUBY_VERSION: 2.7.1
+  NODE_VERSION: 12
+  RAILS_VERSION: 6.0.3
```

Comment:
@hoangmirs Should we increase this one to 6.1.0 as the template has updated to Rails 6.1.0 in #263